app/controllers/ToDo/ShowController.js

```diff
@@ -34,18 +32,20 @@ class ShowController {
      *
      *   @apiError (404) Not Found    The <code>id</code> of the ToDo element was not found.
      */
-    async invoke(request, response, next) {
+    constructor(todoRepository) {
+        this.todoRepository = todoRepository;
+    }
+
+    async invoke(request, response) {
         const todoId = request.params.id;
 
-        Todo.findByPk(todoId)
-            .then(todo => {
-                if (!todo) {
-                    return response.sendStatus(StatusCodes.NOT_FOUND);
-                }
+        const todo = await this.todoRepository.findById(todoId);
+
+        if (!todo) {
+            return response.sendStatus(StatusCodes.NOT_FOUND);
+        }
 
-                return response.json(todo);
```

Comment:
shouldnt be there check if todo belongs to logged user?

Reply:
I think it should. I will change this everywhere , in next pr